packages/serial/src/parsers/SerialAPIParser.ts

```diff
@@ -44,6 +44,12 @@ export class SerialAPIParser extends Transform {
 						this.push(MessageHeaders.ACK);
 						break;
 					}
+					case 0x86: {
+						// This is _maybe_ a corrupted ACK byte from a ZST39 after a soft reset, transform it and keep it
+						this.logger?.ACK("inbound");
+						this.push(MessageHeaders.ACK);
+						break;
+					}
```

Comment:
GROSS. See [issue](https://github.com/zwave-js/node-zwave-js/issues/6399) for details. This probably shouldn't be committed here, I'm happy to back it out.